src/test/java/com/google/api/codegen/testdata/go_library.baseline

```diff
@@ -159,15 +127,15 @@ func NewClient(ctx context.Context, opts ...gax.ClientOption) (*Client, error) {
 
 // Close closes the connection to the API service. The user should invoke this when
 // the client is no longer required.
-func (c *Client) Close() error {
+func (c *LibraryClient) Close() error {
     return c.conn.Close()
 }
 
 // Path templates.
 
 // ShelfPath returns the path for the shelf resource.
-func ShelfPath(shelf string) string {
-    path, err := shelfPathTemplate.Render(map[string]string{
+func (c *LibraryClient) ShelfPath(shelf string) string {
```

Comment:
Yeah that makes sense, not sure why I didn't think that was a reasonable solution.
Will do.